### DISCUSSION TITLE
Enhancement Request: Support for Dynamic Styles in Bubble-Card (Icons and Background Colors)

### DISCUSSION THREAD

**Discussion by JefNewTech** [Clooos/Bubble-Card#732] [Questions about config, custom styles and templates]
Hello,

First, I want to express my appreciation for the amazing work you've done with bubble-card. It's a very flexible and visually appealing card that I've incorporated into my Home Assistant dashboard.

However, I've encountered an issue with handling dynamic styles. I am trying to create a single bubble-card that adjusts both the icon and the background color based on the state of an entity (sensor.next_recycle_event). Unfortunately, neither the icons nor the background colors change dynamically as expected.

I've tried several configurations using JavaScript to dynamically adjust these styles, but it doesn't seem to work. The background remains static, and the icon doesn't change, even though the entity's state is correct.

I’m wondering if this is a current limitation of bubble-card. Would it be possible to enhance support for these dynamic features in a future update?

Thank you again for your ongoing work and the support you provide to the community.

Best regards,

**Comment by Clooos** [comment]
Hi, have you checked the Templates section in the documentation? Because I've added many examples on how you can achieve that.

**Comment by JefNewTech** [comment]
I wasn't paying attention to the example. 
Taking the time to reread my code, I realized that I had a feature that prevented the function 
I close the discussion. 
Thanks for your follow up and thanks again for this wonderful work.

**Reply by Clooos** [reply]
Glad that it's working now, and you're welcome!
